find_strategy_endpoint.py

```
@@ -0,0 +1,63 @@
+import requests
+import json
+
+# Login
+url = "https://cryptouniverse.onrender.com/api/v1/auth/login"
+payload = {
+    "email": "admin@cryptouniverse.com",
+    "password": "AdminPass123!"
+}
+
+response = requests.post(url, json=payload)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)